app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/settings/DeveloperOptionsView.java

```diff
@@ -200,6 +210,18 @@ private void setMultiprocess(boolean value, boolean doApply) {
         }
     }
 
+    private void setPerformance(boolean value, boolean doApply) {
+        mPerformanceSwitch.setOnCheckedChangeListener(null);
+        mPerformanceSwitch.setValue(value, false);
+        mPerformanceSwitch.setOnCheckedChangeListener(mPerformanceListener);
+
+        SettingsStore.getInstance(getContext()).setPerformanceMonitorEnabled(value);
+
+        if (doApply) {
```

Comment:
This block could probably be removed to if calling setting directly from the activity.

Reply:
You might also want to rebase with https://github.com/MozillaReality/FirefoxReality/pull/1403.